scripts/Dockerfile

```diff
@@ -13,16 +18,45 @@ RUN add-apt-repository ppa:rock-core/qt4 -y && \
 RUN rm -rf /catkin_ws && \
     mkdir -p /catkin_ws/src && \
     cd /catkin_ws && \
-    catkin init
+    catkin init && \
+    cd /catkin_ws/src && \
+    git clone https://github.com/RUFFY-369/CrazyS.git && \
```

Comment:
- Why to use your forked repos instead of official ones? What are the changes? 
- Do we need to build it from source?
- Please, move them to the `.rosinstall` file in order to save docker layers

Reply:
- yes, changes in CMakeslists.txt for solving rotors_control build error so official ones weren't used for both repo. The error is as follows:
![rotors_control_build_err](https://user-images.githubusercontent.com/66624139/179526152-f0e18cd7-0d8f-4763-b26c-4b7f9ceac5d5.png)
Also, the updated repos can be pushed in the drones repo in the form of drones' assets

- the source files are cloned in src folder and build is done from workspace root
- done